static/app/components/stream/group.tsx

```diff
@@ -412,7 +412,7 @@ function BaseGroupRow({
     <Wrapper
       data-test-id="group"
       data-test-reviewed={reviewed}
-      onClick={displayReprocessingLayout ? undefined : wrapperToggle}
+      onClick={displayReprocessingLayout || !canSelect ? undefined : wrapperToggle}
```

Comment:
This was an existing bug - if you clicked on an issue stream with `canSelect: false` it would toggle the selected state